tests/integration/states/test_file.py

```diff
@@ -62,6 +62,8 @@
 
 DEFAULT_ENDING = salt.utils.stringutils.to_bytes(os.linesep)
 
+pytestmark = [pytest.mark.skip_on_freebsd(reason="These tests timeout on FreeBSD")]
```

Comment:
```suggestion
pytestmark = [pytest.mark.skip_on_freebsd(reason="These tests timeout on FreeBSD"),]
```

Reply:
Black will then reformat that and turn it into something more readable

Reply:
Black magic exists :smiley: Fixed